src/main/java/com/sourcegraph/cody/PromptPanel.kt

```diff
@@ -119,6 +119,8 @@ class PromptPanel(
     }
   }
 
+  fun focus() = textArea.requestFocusInWindow()
```

Comment:
Is that really needed? Prompt panel has `AncestorListener` listener which focuses on it automatically when it becomes visible.

Reply:
Thanks! I'll verify that

Reply:
@pkukielka I've commented this code and message panel is not always focused (especially when tool window is already opened). I need to add this method.